src/Umbraco.Core/IO/SystemDirectories.cs

```diff
@@ -4,50 +4,50 @@
 namespace Umbraco.Core.IO
 {
     //all paths has a starting but no trailing /
-    public class SystemDirectories
+    public class SystemDirectories : ISystemDirectories
     {
-        public static string Bin => "~/bin";
+        public string Bin => "~/bin";
 
-        public static string Config => "~/config";
+        public string Config => "~/config";
 
-        public static string Data => "~/App_Data";
+        public string Data => "~/App_Data";
 
-        public static string TempData => Data + "/TEMP";
+        public string TempData => Data + "/TEMP";
 
-        public static string TempFileUploads => TempData + "/FileUploads";
+        public string TempFileUploads => TempData + "/FileUploads";
 
-        public static string TempImageUploads => TempFileUploads + "/rte";
+        public string TempImageUploads => TempFileUploads + "/rte";
 
-        public static string Install => "~/install";
+        public string Install => "~/install";
 
-        public static string AppCode => "~/App_Code";
+        public string AppCode => "~/App_Code";
 
-        public static string AppPlugins => "~/App_Plugins";
+        public string AppPlugins => "~/App_Plugins";
 
-        public static string MvcViews => "~/Views";
+        public string MvcViews => "~/Views";
 
-        public static string PartialViews => MvcViews + "/Partials/";
+        public string PartialViews => MvcViews + "/Partials/";
 
-        public static string MacroPartials => MvcViews + "/MacroPartials/";
+        public string MacroPartials => MvcViews + "/MacroPartials/";
 
-        public static string Media => Current.IOHelper.ReturnPath("umbracoMediaPath", "~/media");
+        public string Media => Current.IOHelper.ReturnPath("umbracoMediaPath", "~/media");
```

Comment:
This is going to get ugly. If IOHelper requires ISystemDirectories then ISystemDirectories is going to require `IIOHelper`. We'll need to sort this out before we can continue and mark anything as up for grabs.